.github/workflows/azure-static-web-apps-ambitious-bush-0cbc8790f.yml

```
@@ -1,17 +1,20 @@
+  #   branches:
+  #     - main
+  # pull_request:
+  #   types: [opened, synchronize, reopened, closed]
+  #   branches:
+  #     - main
+  workflow_dispatch:
 
 jobs:
   build_and_deploy_job:
-    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.action != 'closed')
+    if: false # Disabled: prevent Azure Static Web Apps deploy job from running automatically
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)